### ISSUE TITLE
Layer name hidden1 cannot be found in the Keras lab

### ISSUE THREAD

**Issue by deniskrut** [udacity/sdc-issue-reports#204]
Assert was failing for me in Keras lab, because layer named hidden1 did not exist. Instead layers are named dense_input_1 and dense_2 for me.

Here is my code:

model = Sequential()
model.add(Dense(128, input_shape=(32*32*3,)))
model.add(Dense(43))
model.add(Activation('softmax'))

**Comment by deniskrut** [comment]
Here are correct asserts:

assert(model.get_layer(name="dense_input_1").input_shape == (None, 32*32*3)), "The input shape is: %s" % model.get_layer(name="hidden1").input_shape
assert(model.get_layer(name="dense_2").output_shape == (None, 43)), "The output shape is: %s" % model.get_layer(name="output").output_shape

**Comment by avs20** [comment]
Yeah, naming layers was also what I found confusing and cannot find the code for it quickly. 
You can name your layers in keras as 

`model.add(Dense(128,input_shape=(32*32*3,), name="hidden1"))`

**Comment by domluna** [comment]
Fixed in the latest notebook, the test is more robust, doesn't check specific names. Thanks for the report!
